### ISSUE TITLE
Add option to automatically decompress request body

### ISSUE THREAD

**Issue by dee-see** [caido/caido#343]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

This is more frequently seen in responses, but sometimes request bodies can be compressed before being sent to the server with a `Content-Encoding` header.

### Use cases

Using Caido to proxy a desktop thick client's traffic that compresses its request before sending them

**Comment by Sytten** [comment]
Good point, we decompress the responses but not the requests. Another thing to add to the proxy pipeline v2 😅

**Comment by LucianoHanna** [comment]
Any updates?
